todo-mvc/typings.json

```diff
@@ -3,7 +3,8 @@
 	"dependencies": {
 		"@reactivex/RxJS": "npm:@reactivex/rxjs",
 		"immutable": "npm:immutable",
-		"maquette": "npm:maquette"
+		"maquette": "npm:maquette",
+		"node-uuid": "registry:npm/node-uuid#1.4.7+20160723033700"
```

Comment:
I did initially use `@types/node-uuid` and it made a complete mess and caused lots of problems in clashes with existing typings.